src/corelibs/U2Core/src/util/MSAUtils.cpp

```diff
@@ -507,4 +507,14 @@ void MSAUtils::removeColumnsWithGaps(MultipleSequenceAlignment &msa, int require
     }
 }
 
+QString MSAUtils::rollMsaRowName(const QString &rowName, const QSet<QString> &usedRowNamesSet, const QString &suffixSeparator) {
```

Comment:
The similar function `GUrlUtils::rollFileName` already exists. This one could be removed

Reply:
These 2 functions are very different semantically and internally. 

* rollFileName: checks that the file with the given name exists and the result name is a valid file name.
* rollMsaRowName: has nothing to do with files but must provide a valid MSA row name.